### PR TITLE
Flats controller

### PR DESCRIPTION
**Pull request by Rob-Pickard**
Added  flats controller and a set of flat seeds, along with default user. Added migration file to add first and last name for user.

Please migrate and seed database.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,3 +28,6 @@ group :development, :test do
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'dotenv-rails'
 end
+
+# Custom gems
```
Comment (benmclaren) [inline Gemfile]: good practice

File: app/views/flats/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="card">
```
Comment (benmclaren) [inline app/views/flats/index.html.erb]: Card div should be inside the .each

File: Gemfile (hunk 1 above)
Comment (Rob-Pickard) [reply Gemfile]: 👌

File: app/views/flats/index.html.erb (hunk 2 above)
Comment (Rob-Pickard) [reply app/views/flats/index.html.erb]: 😯👍

**Pull request merged**